cmd/harmony/flags.go

```diff
@@ -2115,3 +2111,70 @@ func applyGPOFlags(cmd *cobra.Command, cfg *harmonyconfig.HarmonyConfig) {
 		cfg.GPO.BlockGasLimit = cli.GetIntFlagValue(cmd, gpoBlockGasLimitFlag)
 	}
 }
+
+// cache config flags
+var (
+	cacheDisabled = cli.BoolFlag{
+		Name:     "cache.disabled",
+		Usage:    "Whether to disable trie write caching (archive node)",
+		DefValue: defaultCacheConfig.Disabled,
```

Comment:
so default will be enabled for archival ?

Reply:
ok. that cache configuration doesn't determine whether the node will act as archive node or not right ?

Reply:
> ok. that cache configuration doesn't determine whether the node will act as archive node or not right ?

Correct 👍